components/combobox/common/trigger/index.tsx

```diff
@@ -1,23 +1,40 @@
-import Icon from '~/components/icon'
 import { InputProps } from '~/components/Input/common/input'
 import InputBox from '~/components/Input/input-box'
 
 import useCombobox from '../../use-combobox'
 
-import { TriggerWrapper } from './index.style'
+import { TriggerIcon, TriggerWrapper } from './index.style'
 
 export type ComboboxTriggerProps = InputProps
 
 const Trigger = (props: ComboboxTriggerProps) => {
-  const { open, setOpen } = useCombobox()
+  const { open, setOpen, selected } = useCombobox()
+
+  const openMenu = () => {
+    if (props.disabled) return
+
+    setOpen(() => true)
```

Comment:
아래의 코드랑 무슨 차이로 동작하나요?

```suggestion
    setOpen(true)
```

Reply:
복붙하다 잘못 들어간거 같네요! 수정하겠습니다!